resolwe/permissions/permissions.py

```diff
@@ -39,7 +39,7 @@ def has_object_permission(self, request, view, obj):
             return True
 
         # `share` permission is required for editing permissions
-        if 'permissions' in (view.action or []):
+        if 'permissions' in (view.action or []):  # pylint: disable=superfluous-parens
```

Comment:
This won't work, because `action` is a member variable, not dictionary key. This seams to work:
```
if 'permissions' in view.action:
```